creator-node/src/routes/files.js

```diff
@@ -317,6 +319,13 @@ const getDirCID = async (req, res) => {
 }
 
 module.exports = function (app) {
+  app.get('/processing_status', handleResponse(async (req, res) => {
```

Comment:
how do you feel about /track_content_status?

so like per my other comment

/track_content
/track_content_async
/track_content_status

Reply:
I like a GET route with `/track_content`. will update